src/docfx/Errors.cs

```diff
@@ -95,5 +95,11 @@ public static Error RedirectionDocumentIdConflict(IEnumerable<Document> redirect
 
         public static Error GitShadowClone(string repoPath)
             => new Error(ErrorLevel.Error, "git-shadow-clone", $"Does not support git shallow clone: '{repoPath}'");
+
+        public static Error NullPropertyValue(int startLine = 0, int startColumn = 0)
+            => new Error(ErrorLevel.Warning, "null-property", $"(Line: {startLine}, Column: {startColumn}) Property value is null");
```

Comment:
>value [](start = 116, length = 5)

is it possible to print out the property name?

Reply:
yes, it can be added